IHTSDO/snowstorm
# CDI-28 - Return Object instead of String for concrete values

CDI-28 is concerned with updating the response of Snowstorm to display concrete values as Objects rather than Strings. 

When requesting data of a normal Relationship, the fields "destinationId" and "target" will be populated. However, when requesting data of a concrete Relationship, the field "concreteValue" will replace both "destinationId" and "target". 

To test this:

- Ensure you have imported concrete data
- Navigate to '/$branch/relationships/$relationshipId' for both normal and concrete Relationships. You will either see "destinationId" and "target" fields or "concreteValue" fields.
- Navigate to '/$branch/concepts/$conceptId' for both normal and concrete Relationships. You will either see "destinationId" and "target" fields or "concreteValue" fields.
